internal/versions/versions.go

```diff
@@ -338,6 +358,83 @@ func (h *handler) validateVersions() error {
 	return nil
 }
 
+func existAlready(db *gorm.DB, openShiftReleaseImage common.OpenShiftReleaseImage) bool {
```

Comment:
nit: I think the right spelling is `existsAlready`

Reply:
Thanks